src/components/InputField/index.js

```diff
@@ -10,28 +10,47 @@ import ErrorText from "../ErrorText/index";
 import HintText from "../HintText/index";
 import Input from "../Input/index";
 
-const InputField = ({ children, errorText, hintText }) => (
-  <Label errorText={errorText}>
-    <LabelText errorText={errorText}>{children}</LabelText>
-    {hintText ? <HintText>{hintText}</HintText> : <span />}
-    {errorText ? (
-      <ErrorText errorText={errorText}>{errorText}</ErrorText>
-    ) : (
-      <span />
-    )}
-    <Input type="text" errorText={errorText} />
+const InputField = props => (
```

Comment:
I know I may have written this, but looking at it now I think we should deconstruct props in to meta, children, hint and input, so that it's clear what props are used